.pre-commit-config.yaml

```diff
@@ -21,6 +21,7 @@ repos:
     hooks:
     -   id: isort
         additional_dependencies: [tomli] # to read config from pyproject.toml
+        args: ["--profile", "black"]
```

Comment:
no need to repeat profile=black  here: already in the isort config

Reply:
well I am getting double replacements locally without it....

Reply:
seems to work now ... no idea

Reply:
seems to work now, no idea